src/ripple/test/jtx/impl/Env.cpp

```diff
@@ -50,17 +50,41 @@ namespace test {
 
 namespace jtx {
 
+Env::AppBundle::AppBundle(Application* app_)
+    : app (app_)
+{
+}
```

Comment:
The `Application` object is used in `Env::app()`. The `AppBundle` private class is necessary to allow construction in ctor-initializer lists.